### ISSUE TITLE
How I can allow Session?

### ISSUE THREAD

**Issue by guidiego** [request/request-promise#129]
I want to know if have something like ajax `Access-Control-Allow-Credentials` to pass session across Request :)

Thx!

**Comment by analog-nico** [comment]
tl;dr: Yes, it is possible.

This is more a question you should ask the [Request community](https://gitter.im/request/request) since Request-Promise extends Request by promise capabilities and Request itself is the library that implements the support of the feature your are asking for.

Anyways, I try to answer your question as best as I can:

- Apparently, there is an undocumented option you have to set for your requests to your server: `withCredentials: true`
- You have to create a new cookie jar with `var j = rp.jar()` that you will use in all your requests that belong together by passing the option `jar: j`. See [Request's example section](https://github.com/request/request#examples).

**Comment by guidiego** [comment]
Thx @analog-nico!

**Comment by analog-nico** [comment]
You are welcome!
